pkg/cmd/generate-bazel-extra/main.go

```diff
@@ -38,7 +41,32 @@ func runBuildozer(args []string) {
 	}
 }
 
-func main() {
+func getTestTargets(testTargetSize string) ([]string, error) {
+	match, _ := regexp.MatchString("\\b(?:small|medium|large|enormous)\\b", testTargetSize)
```

Comment:
Nit: Using `regexp` for this is pretty odd. I'd say factor out `testSizeToDefaultTimeout` so it is a *global* variable, and then you can do

```
if _, ok := testSizeToDefaultTimeout[testTargetSize]; !ok {
    ....
}
```